sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1504,6 +1501,7 @@ def task_worker(self):
 
                         if not need_update:
                             # No application updates
+                            self.post_port_active_apsel_to_db(api, lport, host_lanes_mask)
```

Comment:
@mihirpat1 explain in the comment why we need to publish even if Xcvrd is not selecting new application

Reply:
@prgeor I have added a comment now to explain this.